services/web-app/pages/samples/protectedPageWithSSR.js

```diff
@@ -19,14 +19,11 @@ const ProtectedPageWithSSR = (props) => {
   )
 }
 
-export const getServerSideProps = getPropsWithAuth(
-  validUserAuthorizationChecker,
-  async (/* context */) => {
-    // Your normal `getServerSideProps` code here
-    return {
-      props: {}
-    }
+export const getServerSideProps = getPropsWithAuth(isValidIbmUser, async (/* context */) => {
+  // Your normal `getServerSideProps` code here
+  return {
+    props: {}
   }
-)
+})
```

Comment:
One minor thing I just thought of: Can you add the samples folder to the `.dockerignore` file in the web-app dir?

Reply:
Done!